modin/engines/ray/generic/io.py

```diff
@@ -157,9 +157,6 @@ def read_parquet(cls, path, engine, columns, **kwargs):
             for (root, dir_names, files) in os.walk(path):
                 if dir_names:
                     partitioned_columns.add(dir_names[0].split("=")[0])
-                if files:
-                    file_path = os.path.join(root, files[0])
-                    break
```

Comment:
We shouldn't have to continue searching for partitioned_column names once we hit a directory with a file since `os.walk` is a dfs search